tox.ini

```diff
@@ -4,27 +4,29 @@ envlist =
     black
     flake8
     isort
-    py35-django111
-    py{36,37,38}-django{111, 22, 30, master}
+    py{36,37,38,39}-dj{22, 30, 31, master}
     docs
 
-[travis]
+[gh-actions]
 python =
     3.5: py35
     3.6: py36
     3.7: py37
-    3.8: py38
+    3.8: py38, black, flake8, isort
+    3.9: py39
 
 [testenv]
 deps =
-    django111: Django~=1.11.0
-    django22: Django>=2.2<3.0
-    django30: Django>=3.0<3.1
-    djangomaster: https://github.com/django/django/archive/master.tar.gz#egg=django
+    dj111: Django~=1.11.0
```

Comment:
Django 1.11 is EOL and I think should be dropped from the support matrix in any new releases?

```suggestion
```

Reply:
Right, that's actually a leftover anyways, wasn't used in the test matrix anymore.